superdesk/tests/steps.py

```diff
@@ -583,8 +583,7 @@ def fetch_from_provider(context, provider_name, guid, routing_scheme=None, desk_
     else:
         rule_set = None
 
-    provider_service = registered_feeding_services[provider['feeding_service']]
-    provider_service = provider_service.__class__()
+    provider_service = registered_feeding_services[provider['feeding_service']]()
```

Comment:
imo would be good to have some `get_feeding_service(name)` method that would return the instance, so we don't have to bother whenever we need it